### PR TITLE
Adding dockerfile

### PR DESCRIPTION
**Pull request by tomahawk28**
Adding dockerfile for helm 2to3 image. 

Preparing steps for helm3 migration.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
```
Comment (rossf7) [inline Dockerfile]: Why use ubuntu and with no tag so latest gets used? The base image will be huge.

Can you use alpine like we do in draughtsman? https://github.com/giantswarm/draughtsman/blob/bb799c631f8d725c5df104584c8df05422413532/Dockerfile

Comment (tomahawk28) [reply Dockerfile]: I was following it from docker-helm, didn't know that ubuntu with a risk of great size.

Comment (rossf7) [reply Dockerfile]: Right sorry giving docker-helm as an example was a bad idea.

Ubuntu is 25.9 MB vs 2.7 MB for alpine. A smaller base image helps with the china image pulling bs.

https://hub.docker.com/_/ubuntu?tab=tags
https://hub.docker.com/_/alpine?tab=tags

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.10
```
Comment (rossf7) [inline Dockerfile]:
```suggestion
FROM alpine:3.11
```

Much better but 3.11 is out so we should use that.

**Pull request merged**